solutions/client-setup/Kptfile

```diff
@@ -0,0 +1,16 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: client-setup
+  annotations:
+    config.kubernetes.io/local-config: "true"
+info:
+  description: |
+    Landing zone v2 subpackage.
+    Depends on core-landing-zone
+
+    Package to setup a client's namespaces, management project and root sync
```

Comment:
Should we add that it is also deploying the initial client folder

Reply:
done